Other/BackendDocumentation/SQL/SQL Statements/TableCreation.md

```diff
@@ -0,0 +1,125 @@
+# MealDealz Schema V2
+
+## Deals Data
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'deals_data'),
+                BEGIN
+                    CREATE TABLE deals_data (
+                        store VARCHAR(25) PRIMARY KEY,
+                        foodName VARCHAR(50),
+                        saleDetails VARCHAR(255),
+                    )
+                END
+```
+
+## Scraped Times
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'scraped_time'),
+                BEGIN
+                    CREATE TABLE scraped_time (
+                        store VARCHAR(25) PRIMARY KEY,
+                        time DATETIME,
+                    )
+                END
+```
+
+## User Data
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_data')
+                BEGIN
+                    CREATE TABLE user_data (
+                        UserName VARCHAR(25) PRIMARY KEY,
+                        FirstName VARCHAR(30),
+                        LastName VARCHAR(30),
+                        Email VARCHAR(50),
+                        Password VARCHAR(25),
+                        DateJoined DATE,
+                    )
+                END
+```
+
+## User Lists
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_lists')
+                BEGIN
+                    CREATE TABLE user_lists (
+                        UserName VARCHAR(25) PRIMARY KEY,
+                        FoodName VARCHAR(30),
+                        FoodType VARCHAR(30),
+                        Quantity INT,
+                    )
+                END
+```
+
+## User Cookies
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_cookies')
+                BEGIN
+                    CREATE TABLE user_cookies (
+                        UserName VARCHAR(25) PRIMARY KEY,
+                        Cookie VARCHAR(255),
+                    )
+                END
+```
+
+## User Ingredients
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_ingredients')
+                BEGIN
+                    CREATE TABLE user_ingredients (
+                        UserName VARCHAR(25),
+                        FoodName VARCHAR(25),
+                        FoodType VARCHAR(25),
+                        Quantity INT,
+                        PRIMARY KEY (UserName, FoodName)
+                    )
+                END
+```
+
+## User Recipes
+
+```SQL
+IF NOT EXISTS (SELECT * FROM dbo.tables WHERE name = 'user_recipes')
+                BEGIN
+                    CREATE TABLE user_recipes (
+                        RecipeID INT PRIMARY KEY IDENTITY(1,1),
+                        Title VARCHAR(25),
+                        Ingredients VARCHAR(255),
```

Comment:
I found one set of ingredients in the json file that is 970 characters just from scrolling, and where the instructions are 6819 characters

Reply:
I thinks ingredients should be 1500 and instructions probably 10000